include/aws/http/private/http_impl.h

```diff
@@ -46,6 +46,9 @@ enum aws_http_header_name {
     /* Response pseudo-headers */
     AWS_HTTP_HEADER_STATUS,
 
+    /* Cookie header */
+    AWS_HTTP_HEADER_COOKIE,
+
     /* Regular headers */
```

Comment:
trivial: this can just go with the rest of the /* Regular headers */ it doesn't need a comment of its own